src/main/webapp/search/Search.js

```diff
@@ -6,6 +6,7 @@ import ProjectAutocomplete from "../util/ProjectAutocomplete";
 import SearchResults from "./SearchResults";
 import UserAutocomplete from "../util/UserAutocomplete";
 import { UrlConstants } from "../util/UrlConstants";
+import "./style.css";
```

Comment:
@jaime-bernales-broad This is one example of where shared css is a maintenance headache. This PR is about converting the top nav bar to react. However, this addition means that I, as a reviewer, now need to look at the search page to ensure that there are no UI bugs introduced by the addition of this css file. It's unlikely that there is, but still, it complicates both the development and review processes with extra time spent reviewing unrelated code. 

The other example is in `style.css` itself. Due to the number of deletions in that file, both the developer and the reviewer need to look at. I can find usages of styles in that file in quite a few places so each of them would need to be reviewed to ensure no bugs were introduced.

Reply:
Thank you @vvicario. I do understand that the additional styling is necessary in this file - I'm not suggesting that it isn't necessary. In this comment, I'm describing two ways in which a shared css approach is a problematic one. This is also in response to a slack conversation Jaime and I were having about this subject.